e2e/testdata/fn-eval/privilege-options/mount/.expected/exec.sh

```diff
@@ -0,0 +1,22 @@
+#! /bin/bash
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+kpt fn eval \
+--image gcr.io/kpt-fn/kubeval:v0.1 \
+--as-current-user \
+--mount type=bind,src=$(pwd)/schema,dst=/schema-dir \
```

Comment:
how about is the json file? Consider curling it instead of checking it into the repo.

Reply:
If we download it what if the website is down or the url is changed?

Reply:
you already have that problem with the test case that requires --network

Reply:
The schema is only for configmap, so size is small, so having it the repo makes sense. Also, we should avoid dependency on network resources as much as possible (our images itself comes from network I know :))

Reply:
I have changed to pulling schema from network. I agree this this trivial so I think either method should work.

Reply:
Its fine for now.